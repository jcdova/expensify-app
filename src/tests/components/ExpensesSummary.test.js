import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should view 1 expenses totalling correctly', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('should view 2 expenses totalling correctly', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={235987868768} />);
    expect(wrapper).toMatchSnapshot();
});

