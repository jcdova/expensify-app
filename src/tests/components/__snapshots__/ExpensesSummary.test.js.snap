// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should view 1 expenses totalling correctly 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should view 2 expenses totalling correctly 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $2,359,878,687.68
  </h1>
</div>
`;
